src/doc/snabbnfv-test-suite-walk-though.md

```diff
@@ -0,0 +1,163 @@
+# Snabb NFV Test Suite Walk-through
+
+
+## Prerequisites
+
+On a machine with an *Intel 82599 10-Gigabit network interface card* and
+*Docker* installed:
+
+```
+$ lspci | grep 82599
+01:00.0 Ethernet controller: Intel Corporation 82599ES 10-Gigabit SFI/SFP+ Network Connection (rev 01)
+01:00.1 Ethernet controller: Intel Corporation 82599ES 10-Gigabit SFI/SFP+ Network Connection (rev 01)
+
+$ which docker
+/usr/bin/docker
+```
+
+Clone the Snabb Switch repository and build Snabb Switch:
+
+```
+$ git clone https://github.com/snabbnfv-goodies/snabbswitch.git
+[...]
+$ cd snabbswitch
+$ git submodule update --init
```

Comment:
The master branch has no git submodules anymore so this command (`git submodule update --init`) should not be needed. (I know there is currently a submodule on the IPsec branch and that is something we need to think about i.e. do we keep it as a submodule for now, or convert to subtree, or replace that dependency with dynasm code).